m4/sage_python_package_check.m4

```diff
@@ -75,10 +75,13 @@ AC_DEFUN([SAGE_PYTHON_PACKAGE_CHECK], [
         PYTHONUSERBASE="${HOME}/.sage/local"
       ])
 
+      dnl double-quote SAGE_PKG_VERSPEC because platform-specific
+      dnl dependencies like python_version<'3.11' will have single
+      dnl quotes in them.
```

Comment:
Actually, you can use double quotes in install-requires.txt.

And in fact double quotes are better because then `m4/sage_spkg_versions_toml.m4` will be generated correctly!

Reply:
(relevant for #36951)

Reply:
Ok, double quotes it is. I changed the package-check macro to normalize them (back to single quotes) so that both are supported, at least in that one macro.